cfme/fixtures/v2v.py

```diff
@@ -44,23 +50,29 @@ def extract_tag(tag):
 
     if len(tags_set) < 2 or not tags_set.issubset(valid_tags):
         host_obj.remove_tags(tags=tags)
-        return True
-    return False
+        return False
```

Comment:
Its not clear why this boolean is switched, because its not actually clear what this return is supposed to represent :)

Add a bit more to the docblock description as to what this boolean is supposed to represent, so that the intent is clear to the caller.